docs/robots.txt

```diff
@@ -0,0 +1,4 @@
+User-agent: *
+Allow: /*/stable/
+Allow: /en/stable/   # Fallback for bots that don't understand wildcards
```

Comment:
I don't understand robots at all, but just wonder whether this would also let the latest dev docs, `/en/latest/` to be found, too?
@astrofrog - we would like that, too and only would like to avoid the old versions to be picked up?

Reply:
@bsipocz thanks. You are right, it would not. For that I'd add `/*/latest/` and `/en/latest/`as allowed paths. Is that desired?

Reply:
Added `latest` to Allowed paths.